src/plugins/data/common/index_patterns/types.ts

```diff
@@ -51,3 +52,46 @@ export interface IndexPatternAttributes {
 
 export type OnNotification = (toastInputFields: ToastInputFields) => void;
 export type OnError = (error: Error, toastInputFields: ErrorToastOptions) => void;
+
+export interface FieldFormatSpec {
+  id: string;
+  params: IFieldFormatMetaParams;
+}
+
+export type AggregationRestrictions = Record<
+  string,
+  {
+    agg?: string;
```

Comment:
are these stored with index pattern (rollups) ? or are they retrieved on the fly ?

Reply:
yes, just moved